tests/common.go

```diff
@@ -9500,3 +9512,24 @@ func GenerateS3BucketPolicy(sid string, encryptionPolicy string, bucketName stri
 
 	return policy, nil
 }
+
+// GetSseS3EncryptionType fetches SSE type for S3 bucket from the environment variable
+func GetSseS3EncryptionType() (api.S3Config_Sse, error) {
+	var sseType api.S3Config_Sse
+	s3SseTypeEnv := os.Getenv("S3_SSE_TYPE")
+	if s3SseTypeEnv != "" {
+		sseDetails, err := s3utils.GetS3SSEDetailsFromEnv()
+		if err != nil {
+			return sseType, err
+		}
+		switch strings.ToUpper(string(sseDetails.SseType)) {
```

Comment:
There is no need to convert it to upper case or string

Reply:
done